homeassistant/components/profiler/__init__.py

```diff
@@ -254,6 +257,19 @@ async def _async_dump_scheduled(call: ServiceCall) -> None:
             arepr.maxstring = original_maxstring
             arepr.maxother = original_maxother
 
+    async def _async_asyncio_debug(call: ServiceCall) -> None:
+        """Enable or disable asyncio debug."""
+        enabled = call.data[CONF_ENABLED]
+        # Always log this at critical level so we know when
+        # it's been changed when reviewing logs
+        _LOGGER.critical("Setting asyncio debug to %s", enabled)
+        # Make sure the logger is set to at least INFO or
+        # we won't see the messages
+        base_logger = logging.getLogger()
+        if base_logger.getEffectiveLevel() > logging.INFO:
```

Comment:
Should we only do this when enabled is true?

Reply:
I was going to say that there is no reason to call the service if its not already enabled, but someone will do that of course 😆 

adjusted